src/shared/services/email/email.service.ts

```diff
@@ -0,0 +1,28 @@
+import { createTransport } from 'nodemailer';
+import { smtpConfig } from '../../configs/index';
+
+export interface IMessageToEmail_Return {
+  from: string;
+  to: string;
+  subject: string;
+  html: string;
+}
+
+export const transporter = createTransport({
```

Comment:
This function should be combined with the one below to create a complete `EmailService.sendEmail` method which can be called in 1 line